CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Variant page crashing for cases with old OMIM terms structure (a list of integers instead of dictionary)
 - Variant page crashing when creating MARRVEL link for cases with no genome build
 - SpliceAI documentation link
+- List gene names densely in general report for SVs that contain more than 3 genes
```

Comment:
Hm, yes, that sounds about right; I think you settled on 5 below though?

Reply:
Right, I've changed to 5 in the end, will fix!